app/src/main/java/com/bytemedrive/customer/entity/Customer.kt

```diff
@@ -7,13 +7,28 @@ data class Customer(
     val username: String,
     val walletId: UUID?,
     val signUpAt: ZonedDateTime?,
-    val balanceGbm: Long?
+    val balanceGbm: Long?,
+    val dataUsageOverWiFi: Boolean,
+    val thumbnailCreation: Boolean? = null,
 ) {
 
     constructor(customerEntity: CustomerEntity) : this(
         customerEntity.username,
         customerEntity.walletId,
         customerEntity.signUpAt,
-        customerEntity.balanceGbm
+        customerEntity.balanceGbm,
+        customerEntity.dataUsageOverWiFi,
+        customerEntity.thumbnailCreation
     )
+
+    companion object {
```

Comment:
This is not good, remove and rework default values in settings screen / view model